src/sentry/issues/search.py

```diff
@@ -227,13 +227,28 @@ def _query_params_for_generic(
 }
 
 
+def _update_profiling_search_filters(
+    search_filters: Sequence[SearchFilter],
+) -> Sequence[SearchFilter]:
+    updated_filters = []
+
+    for sf in search_filters:
+        if sf.key.name in ("notHandled", "error.handled"):
+            updated_filters.append(SearchFilter(SearchKey("1"), "=", SearchValue("2")))
```

Comment:
The resulting snuba query is as follows:
```
 SELECT (group_id AS _snuba_group_id), (uniq(_snuba_group_id) AS _snuba_total), (multiply(toUInt64(max((client_timestamp AS _snuba_timestamp))), 1000) AS _snuba_last_seen)
  FROM search_issues_local SAMPLE 1.0
 WHERE greaterOrEquals(_snuba_timestamp, toDateTime('2022-11-18T17:28:53', 'Universal'))
   AND less(_snuba_timestamp, toDateTime('2023-02-16T17:29:53', 'Universal'))
   AND in((project_id AS _snuba_project_id), tuple(4551438950006785))
   AND has(_tags_hash_map, cityHash64('1=2'))
   AND in(_snuba_group_id, (1, 2, 3))
   AND in(_snuba_project_id, tuple(4551438950006785))
 GROUP BY _snuba_group_id WITH TOTALS
 ORDER BY _snuba_last_seen DESC,
          _snuba_group_id ASC
 LIMIT 3
OFFSET 0
```

There's probably a better way to do a no-op query, or even better not query at all here. We might look to add an early return if certain conditions are met, but for now to quickly fix this issue, we'll add this as a placeholder.

Reply:
Ok, makes sense short term, there's probably some other way to handle this that we can look into